libs/platform/package.json

```diff
@@ -23,6 +23,7 @@
     "url": "https://github.com/SAP/fundamental-ngx"
   },
   "dependencies": {
-    "@fundamental-ngx/core": "VERSION_PLACEHOLDER"
+    "@fundamental-ngx/core": "VERSION_PLACEHOLDER",
+    "xml2js":"0.4.23"
```

Comment:
Can the version be also configured as a placeholder?

Reply:
I did look at `sync-versions.sh` for this but that seems to assume the library is also declared in the root `package.json`, whereas `xml2js` is only declared here. I am not sure how to add the placeholder in this case or if it is needed.